sale_last_price_info/models/product.py

```diff
@@ -0,0 +1,29 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.one
+    def _get_last_sale(self):
+        """ Get last sale price, last sale date and last customer """
+        lines = self.env['sale.order.line'].search(
+            [('product_id', '=', self.id),
+             ('state', 'in', ['confirmed', 'done'])]).sorted(
+            key=lambda l: l.order_id.date_order, reverse=True)
+        self.last_sale_date = lines[:1].order_id.date_order
+        self.last_sale_price = lines[:1].price_unit
+        self.last_customer = lines[:1].order_id.partner_id
+
+    last_sale_price = fields.Float(
+        string='Last Sale Price', compute='_get_last_sale')
+    last_sale_date = fields.Date(
+        string='Last Sale Date', compute='_get_last_sale')
+    last_customer = fields.Many2one(
```

Comment:
Well, I don't recognise that user, but he is saying:
> Same for the variables name. Especially since the new api. If we have a record or recordset, don't name the variable with _id, when I see a _id in a variable name, I expect a number, not a record

So I read exactly the opposite, and why variable names are going to be different in the behaviour from fields if you are using the same notation for accesing both? I see here clearly an inconsistency